build/source/driver/multi_driver.f90

```diff
@@ -147,10 +179,17 @@ real(dp),parameter        :: safety = 0.01_dp               ! safety factor to e
 real(dp),allocatable      :: upArea(:)                      ! area upslope of each HRU
 ! general local variables
 real(dp)                  :: fracHRU                        ! fractional area of a given HRU (-)
+integer(i4b)              :: fileUnit                       ! file unit (output from file_open; a unit not currently used)
+character(LEN=256),allocatable :: dataLines(:)    ! vector of character strings from non-comment lines
```

Comment:
Probably leave for now (i.e. don't change, cause we can do this in a small dedicated PR - I'll open an issue for it)